src/apps/dashboard/routes/_asyncRoutes.ts

```diff
@@ -12,6 +12,7 @@ export const ASYNC_ADMIN_ROUTES: AsyncRoute[] = [
     { path: 'playback/trickplay', type: AppType.Dashboard },
     { path: 'plugins/:pluginId', page: 'plugins/plugin', type: AppType.Dashboard },
     { path: 'tasks', type: AppType.Dashboard },
+    { path: 'tasks/edit', type: AppType.Dashboard },
```

Comment:
We should update this to use a path parameter for the task id

```suggestion
    { path: 'tasks/:id', type: AppType.Dashboard },
```